tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CQyfa3akfIEZ"
+      },
+      "source": [
+        "**Level**: Intermediate\n",
+        "\n",
+        "**Time to complete**: 20 minutes\n",
+        "\n",
+        "**Prerequisites**: Prepare the Colab environment (see links below).\n",
+        "\n",
+        "**Nodes Used**: InMemoryDocumentStore, MultiModalRetriever\n",
+        "\n",
+        "**Goal**: After completing this tutorial, you will have learned about the MultiModalRetriever, and built a simple retrieval pipeline that search relevant images given a text query."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fzn2uA1Be1Km"
+      },
+      "source": [
+        "## Preparing the Colab Environment\n",
+        "\n",
+        "- [Enable GPU Runtime in GPU](https://docs.haystack.deepset.ai/v5.2-unstable/docs/enable-gpu-runtime-in-colab)\n",
+        "- [Check if GPU is Enabled](https://docs.haystack.deepset.ai/v5.2-unstable/docs/check-if-gpu-is-enabled)\n",
+        "- [Set logging level to INFO](https://docs.haystack.deepset.ai/v5.2-unstable/docs/set-the-logging-level)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Hl92D-ZlycPh"
+      },
+      "outputs": [],
+      "source": [
+        "%%bash\n",
+        "\n",
+        "pip install --upgrade pip\n",
+        "pip install farm-haystack[colab]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "s62Z-1xL40vp"
+      },
+      "outputs": [],
+      "source": [
+        "import logging\n",
+        "\n",
+        "logging.basicConfig(format=\"%(levelname)s - %(name)s -  %(message)s\", level=logging.WARNING)\n",
+        "logging.getLogger(\"haystack\").setLevel(level=logging.INFO)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KkVAG7FdXsEU"
+      },
+      "source": [
+        "# Initializing the DocumentStore\n",
+        "\n",
+        "A DocumentStore contains Documents, which in this case are references to the images that Haystack will compare with your query. Here we are using the InMemoryDocumentStore since it requires no external dependencies. To learn more about the DocumentStore and the different types of external databases that we support, see [DocumentStore](https://docs.haystack.deepset.ai/docs/document_store)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "dK86aFlSYQXv",
+        "outputId": "280d53aa-b932-474c-8f39-f246118d5f83"
+      },
+      "outputs": [
```

Comment:
Be sure to clean up all the outputs before committing the final version

Reply:
Done.